hubblestack/hec/obj.py

```diff
@@ -147,6 +147,7 @@ def __init__(self, token, index, http_event_server, host='', http_event_port='80
         self.currentByteLength = 0
         self.server_uri = []
 
+        proxy =  proxy or __opts__.get('https_proxy', None)
```

Comment:
Is calling it `https_proxy` correct here or would it be applied also in case if http scheme is used?

Reply:
@fossam I intended to name it as 'proxy' but it raised a conflict in salt's loader.py. Moreover as of now we are only using this variable in azurefs and splunk which both run on port 443. If in future there's a need for http only proxy, we can add a parameter like 'http_proxy'.

Reply:
Thank you. 
Can the setting and the caveats be documented in some form, say in example configuration file?

Reply:
Sure, I'll add a sample config block and document it as well.